### ISSUE TITLE
Always show tooltips the same place.

### ISSUE THREAD

**Issue by Hapseleg** [PathOfBuildingCommunity/PathOfBuilding#3730]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

When mousing over anything that will show a tooltip (item, gem, passives) the tooltip will often "bounce" from one side to the other (or from top to bottom) when you mouse over the next "thing". This is often worst when it comes to items.

### Describe the solution you'd like

Have the tooltip stay in the same place when mousing over "things" that will have a tooltip pop-up. This could be done by forcing the tooltip to always show up top-right of the mouse (or bottom-left etc).

### Describe alternatives you've considered

Instead of having the item attached to the mouse you could have it show up in a certain location, for example always show up in a pane at the right side.
If you use this solution I would recommend adding an option to enable/disable it as the mouse over thing is something everyone is used to cause its like that in the game :)

### Additional context

To replicate this, set your resolution to 1920x1080 and open up items, Mouse over Abberath's Hooves and Abberath's Horn. The tooltip will jump from top to bottom.
To replicate it where its from left to right go to gems and mouse over for example "Spirit offering" (right) and then "Zealotry" (left).

Bottom:
![image](https://user-images.githubusercontent.com/12915302/140536197-94fe93b6-d57a-4697-a6fb-2dc48e61b2bb.png)

Top:
![image](https://user-images.githubusercontent.com/12915302/140536315-afca0656-37f4-41d6-820b-f9ca71199740.png)

Right:
![image](https://user-images.githubusercontent.com/12915302/140536446-4e545fd0-fda2-4795-a703-ad838fddf9d5.png)

Left:
![image](https://user-images.githubusercontent.com/12915302/140536477-17be31f9-ae9c-40bb-8117-1433736148c6.png)

Extremely quick sketch up of how it could look:
![image](https://user-images.githubusercontent.com/12915302/140537073-a5f4c679-3c9d-43d1-9826-d173d5e1d9c2.png)

**Comment by Wires77** [comment]
Tooltips already do this, technically, but there are a bunch of limitations that make them pop around.

1. Always will appear touching the item that spawned the tooltip in the bottom right
2. If the tooltip is too long to fit on the screen, swap the position to the upper right.
3. If the tooltip is too wide to fit on the screen, swap the position to the lower left (or upper left)
4. Tooltip can never cover the build header or sidebar
5. Tooltip can never cover the text that spawns the tooltip

The best we could do would be to have the tooltip sit on the bottom of the screen instead of popping up above the item, but anything more would probably end up violating one of those rules.

Regarding your last image, that's essentially the view you get when you double-click on an item, so there's no reason to move the tooltip functionality to there.

**Comment by Hapseleg** [comment]
Yeah you are right that there are a lot of rules. I actually like the idea of having the item show up like when you double click it, the tutorial part is as you say it, for new people, so why have it show all the time? Could perhaps add an option to show/hide it? It it a lot of space after all.
